content/logs/languages/php.md

```diff
@@ -0,0 +1,503 @@
+---
+title: PHP log collection
+kind: documentation
+further_reading:
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+- link: "/logs/explore"
+  tag: "Documentation"
+  text: Learn how to explore your logs
+- link: /logs/faq/log-collection-troubleshooting-guide
+  tag: "FAQ"
+  text: Log Collection Troubleshooting Guide
+---
+
+## Overview
+
+Write your PHP logs into a file, then [use the Agent](/logs) to forward them to Datadog, choose between the following libraries:
+
+* [Monolog](/logs/languages/php/#php-monolog)
+* [Symfony](/logs/languages/php/#php-symfony)
+* [Zend-Log](/logs/languages/php/#php-zend-log)
+
+## PHP Monolog
+
+Use Composer to add Monolog as a dependency:  
+ 
+```
+composer require "monolog/monolog"
+```
+
+Alternatively, install it manually:
+
+1. Download Monolog from the repository and include it to the libraries
+2. Initialize the instance in the application's bootstrap
+
+    ```php
+    <?php
+
+    require __DIR__ . '/vendor/autoload.php';
+
+    // load Monolog library
+    use Monolog\Logger;
+    use Monolog\Handler\StreamHandler;
+    use Monolog\Formatter\JsonFormatter;    
+    ```
+
+
+### Setup - Log into a file with Monolog
+
+The following configuration enables the Json formatting and writes the logs and events into the `application-json.log` file. Edit your code, right after the initialization of the Monolog instance and add a new handler:
+
+```php
+ <?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+// load Monolog library
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\Formatter\JsonFormatter;
+
+// create a log channel
+$log = new Logger('channel_name');
+
+// create a Json formatter
+$formatter = new JsonFormatter();
+
+// create a handler
+$stream = new StreamHandler(__DIR__.'/application-json.log', Logger::DEBUG);
+$stream->setFormatter($formatter);
+
+// bind
+$log->pushHandler($stream);
+
+// an example
+$log->info('Adding a new user', array('username' => 'Seldaek'));
+```
+
+Then [stream your log file to Datadog](/logs/#custom-log-collection)
+
+### Add meta field and context
+
+Much of the usefulness information comes from additional context data that you add to your logs and events.  
+Monolog makes this very convenient by providing methods to set thread local context data that is then submitted automatically with all events. At any moment, log an event with a contextual data:
+
+```php
+ <?php
```

Comment:
For the code snippets that are one-liners, I would omit the `<?php` at the top - since it implies it's at the beginning of the file. Instead,  I would write:

> ```php
> $logger->info('Adding a new user', array('username' => 'Seldaek'));
> ```

Reply:
👌